providers/stores/azblob.js

```diff
@@ -33,11 +34,13 @@ class AzBlobStore extends AbstractStore {
           this.containerName,
           name,
           continuation,
+          { include: azure.BlobUtilities.BlobListingDetails.METADATA },
```

Comment:
is there much of a cost to getting the metadata? if it is not trivial, perhaps only add this option if `type === entity`

Reply:
I think we're only getting a few blobs at a time, but it's definetly not free.

I'll see if it works with

```
{ include: type === 'entity' ? azure.BlobUtilities.BlobListingDetails.METADATA : '' },
```

Reply:
since we're going to preserve case if `type == entity || type == result` I will include metadata for both